docs/ProfileDeveloperGuide.md

```diff
@@ -230,6 +231,27 @@ Is satisfied if `field` is null or absent.
 
 Is satisfied if `field` is of type represented by `value` (valid options: `decimal`, `integer`, `string`, `datetime`, `ISIN`, `SEDOL`, `CUSIP`, `RIC`, `firstname`, `lastname` or `fullname`)
 
+<div id="predicate-fromfile"></div>
+
+### `fromFile` _(field, value)_
+
+```javascript
+{ "field": "country", "is": "fromFile", "value": "countries" }
+```
+
+Populates a set from the new-line delimited file, where each line represents a string value to load.
+
+Example file excerpt:
+```javascript
+...
+England
+Wales
+Scotland
+...
+```
+
+After loading the set from the file, this constraint behaves identically to the [inSet](#predicate-inset) constraint.
+
```

Comment:
It would be good to add a short explanation showing where to put the files, and whether file extensions should be included in the name, and which file extensions are usable.